defaults/main.yml

```diff
@@ -66,6 +66,14 @@ mon_icingaweb_ldap_bindpwd: ""
 mon_icingaweb_ldap_roles: []
 
 mon_icingaweb_dbauth: False
+mon_icingaweb_extramodules:
+  - name: elasticsearch
```

Comment:
Would it make sense to package this?

Reply:
Hmm. In theory we could, however, I fail to see an advantage as this doesn't touch anything under apt's control, so it can't conflict with packaging by definition, therefore it would only be overhead...